lib/ui/blocs/deeplink/deeplink_bloc.dart

```diff
@@ -54,4 +93,24 @@ class DeeplinkBloc extends Bloc<DeeplinkEvent, DeeplinkState> {
       );
     }
   }
+
+  Future<void> _incomingESRLink(_IncomingESRLink event, Emitter<DeeplinkState> emit) async {
+    final result = await _parseQRCodeUseCase.run(ParseESRLinkInput(esrLink: event.link));
+    if (result.isValue) {
+      emit(
+        state.copyWith(
+          command: PageCommand.navigateToSignTransaction(result.asValue!.value),
+        ),
+      );
+    } else {
+      print('error: ${result.asError!.error}');

```

Comment:
long helper. If you copied this error printing from somewhere pls fix over there too.

Reply:
Done